titanic.ipynb

```diff
@@ -5,14 +5,14 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 Line 17_
```python
# Create a countplot of the man/woman/child classification
sns.countplot(data=titanic, x='who', hue='alive', palette='bright')

# Add some labels and title
plt.title('Survival Rates by Man/Woman/Child')

# Show the plot
plt.show()
```
Single line comment there

_Respond and view the context [here](https://gitnotebooks.com/smith-kyle/notebook-diff-tests/pull/25)._